components/board/pi/common/common.go

```diff
@@ -17,21 +17,27 @@ var ModelName = resource.NewDefaultModel("pi")
 
 // ServoConfig is the config for a pi servo.
 type ServoConfig struct {
-	Pin      string   `json:"pin"`
-	Min      int      `json:"min,omitempty"`
-	Max      int      `json:"max,omitempty"`
-	StartPos *float64 `json:"starting_position_degs,omitempty"`
-	HoldPos  *bool    `json:"hold_position,omitempty"` // defaults True. False holds for 500 ms then disables servo
+	Pin       string   `json:"pin"`
+	Min       int      `json:"min,omitempty"`
+	Max       int      `json:"max,omitempty"`
+	StartPos  *float64 `json:"starting_position_degs,omitempty"`
+	HoldPos   *bool    `json:"hold_position,omitempty"` // defaults True. False holds for 500 ms then disables servo
+	BoardName string   `json:"board"`
```

Comment:
Does this count as a breaking change, since the config file now has a new required entry? We'll need to write to the "breaking changes" channel on Slack.